Cargo.toml

```
@@ -55,6 +55,7 @@ cmake = "0.1"
 [dependencies]
 log = { version = "0.4", features = ["std"] }
 libc = "0.2"
+libm = "0.1"
```

The latest version of `libm` is `0.2.1` so this should be `0.2`
```suggestion
libm = "0.2"
```